packages/marketplace/src/services/apps.ts

```diff
@@ -0,0 +1,27 @@
+import { fetcher } from '@reapit/elements'
+import { URLS, generateHeader } from '../constants/api'
+
+export interface FetchAppDetailParams {
```

Comment:
You define this interface but don't use it! 

Also, needs a test I think for this file

Reply:
`FetchAppDetailParams` is used in `src/actions/client.ts`
`export const clientFetchAppDetail = actionCreator<FetchAppDetailParams>(ActionTypes.CLIENT_FETCH_APP_DETAIL)`

and in `src/actions/developer.ts`
`export const developerFetchAppDetail = actionCreator<FetchAppDetailParams>(ActionTypes.DEVELOPER_FETCH_APP_DETAIL)`

I added a test for this file by the way